apps/examples/testcase/Kconfig

```diff
@@ -20,6 +20,7 @@ source "$APPSDIR/examples/testcase/ta_tc/audio/utc/Kconfig"
 source "$APPSDIR/examples/testcase/ta_tc/audio/itc/Kconfig"
 source "$APPSDIR/examples/testcase/le_tc/filesystem/Kconfig"
 source "$APPSDIR/examples/testcase/le_tc/kernel/Kconfig"
+source "$APPSDIR/examples/testcase/le_tc/drivers/Kconfig"
```

Comment:
i think you may have to follow alphabetic order ?

Reply:
Is this present in any guidelines?
Alphabetic order has not been followed at several places. Even in the same file.

Reply:
maybe there is no guidelines, but we recommend you to follow the alphabetic order.
yes, you are right. some file doesn't follow,, but from now on, we'd better follow the order.
Thanks.

Reply:
Ok done.